packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.base.tsx

```diff
@@ -69,6 +69,11 @@ export class ChoiceGroupBase extends BaseComponent<IChoiceGroupProps, IChoiceGro
 
     const ariaLabelledBy = label ? this._id + '-label' : (this.props as any)['aria-labelledby'];
 
+    // In cases where no option is checked, set tab index to first enabled option so that ChoiceGroup remains tabbable.
+    // If no options are enabled, ChoiceGroup is not tabbable. If any option is checked, do not set keyDefaultTab.
+    const firstEnabledOption = disabled || options === undefined ? undefined : options.find(option => !option.disabled);
```

Comment:
I think I can pretty easily try `data-is-focusable` here and see if FTZ recognizes it since it's almost a 1:1 substitution. If you know of any other methods, let me know.

Reply:
This is the ARIA example I used as a reference: https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20160317/examples/radio/radio.html

Reply:
I tested `data-is-focusable` and it works! I've switched the approach.